src/main/java/org/jsmart/zerocode/core/di/ApplicationMainModule.java

```diff
@@ -58,7 +65,16 @@ public Properties getProperties(String host) {
         final Properties properties = new Properties();
         try {
             properties.load(getClass().getClassLoader().getResourceAsStream(host));
-
+            /// below code is for backward compatibility, remove this after few releases ///
+            if(properties.get(WEB_APPLICATION_ENDPOINT_HOST) == null){
```

Comment:
A test please proving both works. Again this shd be easy enough, while you did a replace all, keep one of the place as `restful...`, then the test should not fail.
> If you have already taken care, then you can ignore this comment.

Reply:
@authorjapps  added new test case with new property file ,so we can easily remove these two files after backward compatibility feature is removed